libcontainer/cgroups/systemd/common.go

```diff
@@ -158,8 +160,7 @@ func findDeviceGroup(ruleType devices.Type, ruleMajor int64) (string, error) {
 	return "", nil
 }
 
-// DeviceAllow is the dbus type "a(ss)" which means we need a struct
-// to represent it in Go.
+// deviceAllowEntry represents the DeviceAllow dbus type "a(ss)" in Go.
```

Comment:
Hmm, your comment makes less sense than the original.

Perhaps
```suggestion
// deviceAllowEntry is a type representing an entry for DeviceAllow
// dbus property value, which has dbus type of "a(ss)".
```